src/main/java/no/digipost/api/client/DigipostClient.java

```diff
@@ -71,11 +73,12 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(DigipostClient.class);
 
-    private final EventLogger eventLogger;
-    private final ApiService apiService;
-    private final MessageSender messageSender;
-    private final MessageDeliverer deliverer;
-    private final DocumentCommunicator documentCommunicator;
+	private final EventLogger eventLogger;
```

Comment:
Det er for øvrig litt rart, for mergen med master bestod i hovedsak i "accept changes from server" for min del.